daily.sh

```diff
@@ -133,14 +133,27 @@ check_dependencies() {
     local branch ver_56 ver_71 ver_72 ver_73 python3 python_deps phpver pythonver old_branches msg
 
     branch=$(git rev-parse --abbrev-ref HEAD)
-    scripts/check_requirements.py > /dev/null 2>&1 || pip3 install -r requirements.txt > /dev/null 2>&1
+
+    python3=$(python3 -c "import sys;print(int(sys.version_info < (3, 4)))" 2> /dev/null)
```

Comment:
Don't check this in daily.sh, only in validation.

Reply:
I am sorry, I don't get it. In th daily.sh version on master, we already have check_requirepents.py || pip install -r requirements.py
I just split into python2 mode that stays the same, and python3 that adds more checks.
Running pip as user does not require privileges and will install necessary packages.

What does it break exactly ? Sorry if this doesn't make sense to you :(